spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :user do
+    name { 'John Doe' }
+    email { Faker::Internet.unique.email }
+    password { 123_456 }
```

Comment:
We'll have a conflict here. I set mine to be 123456. I guess we can pick the one that requires less changes in tests